package.json

```diff
@@ -187,19 +190,20 @@
         "pretty-bytes": "^6.1.1",
         "pretty-quick": "^4.1.1",
         "re-template-tag": "^2.0.1",
-        "react": "^18.2.0",
-        "react-dom": "^18.2.0",
+        "react": "^18.3.1",
+        "react-dom": "^18.3.1",
```

Comment:
Question - do we need the react dependency for anything other than Storybook? If not, I can remove it from here because Storybook no longer requires React as a peer dep

Reply:
Let me confirm but i believe its used for the react wrappers but ill follow up

Reply:
so looks like `react` and `react-dom` are used by another storybook that exists in `projects/custom-css-vars-viewer`. I don't know if that's even used anymore or if we can remove that project too? Otherwise it appears safe to remove

Also I think we can remove `next` as a package too.

Reply:
I tested out removing next and got a set of these errors so maybe we hold off on that for now:

```
react/icons-ui/next/TripleGripper.ts:13:21 - error TS2307: Cannot find module 'next/dynamic' or its corresponding type declarations.

13 import dynamic from 'next/dynamic';
```